### ISSUE TITLE
Color Discrepancy Issue in RGB SparseCtrl and AnimateDiff v3: Seeking Solutions

### ISSUE THREAD

**Issue by chuck-ma** [guoyww/AnimateDiff#278]
"I'm using RGB SparseCtrl and AnimateDiff v3. The color of the first frame is much lighter than the subsequent frames. Consequently, if I continuously loop the last frame as the first frame, the colors in the final video become unnaturally dark. Is there any solution to this

**Comment by Botoni** [comment]
The same happens to me, and also can be seen in the samples in the github page. There's a color/saturation shift from the reference frame to the following ones.

Here's an example:
![Screenshot from 2024-01-04 23-35-43](https://github.com/guoyww/AnimateDiff/assets/51716817/1e2044bf-7d20-4402-8b4f-12c3a7bc4064)

**Comment by ps3dood** [comment]
yes, seeing this always too. Adding a ColorMatch helps, but is far from perfect
